src/ui/transforms.js

```diff
@@ -40,40 +40,42 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
     };
 
     /* A transform to turn an array into a delimited string
-     * Values can be optionally be
-     * included only if they are string value and are truthy (i.e. not an empty
-     * string). Values can also be accessed via a specific path relative to each
-     * term.
-     * - separator: the string delimiter to be inserted between each term
-     * - stringOnly (optional): flag to exclude all falsy or non-string values
-     * - path (optional): an EL path on each item in the terms collection
+     * Values can also be accessed via a specific object path relative to each term.
+     * - "separator" (optional): the delimiter to be inserted between each term
+     * - "trailingSeparator" (optional): flag to include the separator at the end
+     * - "stringOnly" (optional): flag to allow only non-empty strings
+     * - "path" (optional): an EL path on each item in the terms collection
      */
     fluid.defaults("sjrk.storyTelling.transforms.arrayToString", {
         "gradeNames": [ "fluid.standardTransformFunction", "fluid.multiInputTransformFunction" ],
```

Comment:
This isn't really a multiInputTransformFunction since we don't expect to resolve all of these values from model material dynamically. This can just be a standardTransformFunction with a bunch of options in the transformSpec at top level.